wiki/osu!supporter/de.md

```diff
@@ -1,56 +1,157 @@
+
+- Globale Ranglisten für jede Kombination an [Spielmodifikationen](/wiki/Game_modifier)
+- Länderspezifische Ranglisten von Spielern, die die gleiche Flagge tragen
+- Freundesranglisten, die die Position eines Spielers auf einer Beatmap verglichen mit seinen Freunden zeigen
+
+### Erhöhte Grenzwerte
+
+osu! bietet eine Lockerung der Grenzwerte für verschiedene Online-Features für Unterstützer an:
+
+| Wert | Normales Limit | osu!supporter-Limit |
+| :-- | :-: | :-: |
+| [Ausstehende Beatmaps](/wiki/Beatmap/Category#work-in-progress-and-pending) | `2 + min(gerankte Beatmaps, 6)`, bis zu **8**[^pending-beatmaps-ref] | `8 + min(gerankte Beatmaps, 12)`, bis zu **20**[^pending-beatmaps-ref] |
+| Online-Beatmap-Favoriten | 100 | 1000 |
+| Anzahl der Freunde | 250 | 500 |
+
+Des Weiteren werden Besitzern von osu!supporter weniger strenge Download-Beschränkungen gewährt.
+
+### Extra skinnbare Elemente
+
+Nach Erhalt von osu!supporter können einige visuelle und akustische Elemente der osu!-Benutzeroberfläche benutzerdefiniert angepasst werden:
+
+| Element | Beschreibung |
+| :-- | :-- |
+| `menu-background.jpg` | Der Hintergrund des Hauptmenüs |
+| `welcome_text.png` | Der Text "welcome", der beim Starten des Spiels erscheint |
+| `welcome.wav` | Das Sample "welcome to osu!", das beim Starten des Spiels abgespielt wird |
+| `seeya.wav` | Das Sample "see ya next time", das beim Beenden des Spiels abgespielt wird |
+
+Für mehr Details, siehe [Skinning/Benutzeroberfläche § Hauptmenü](/wiki/Skinning/Interface#main-menu) und [Skinning/Sounds § Hauptmenü](/wiki/Skinning/Sounds#main-menu).
+
+### Erweiterte Suche
+
+::: Infobox
+![](img/beatmap-search-DE.png "Beatmap-Suchseite auf der Webseite, mit zwei neuen Filtern am unteren Ende")
+:::
+
+Unterstützer haben Zugang zu einem erweiterten Satz von [Beatmap-Suchfiltern](https://osu.ppy.sh/beatmapsets):
+
+- Beatmaps, die gespielt (oder nicht gespielt) wurden
+- Beatmaps, auf denen ein Score mit einer bestimmten [Note](/wiki/Gameplay/Grade) erreicht wurde
+
+### Mehrspieler in experimentellen Builds von osu!
+
+Unterstützer können auf der Early-Access-Updatequelle "cutting edge" auf den Mehrspielermodus zugreifen.
+
+## Verbleibende Dauer
+
+::: Infobox
+![](img/status-DE.jpg?1 "osu!supporter-Kaufstatistik")
+:::
+
+Die restliche osu!supporter-Dauer sowie die Gesamtzahl an Beiträgen, Tag-Käufen und Geschenken ist oben auf der [osu!supporter-Promo-Seite](https://osu.ppy.sh/home/support) verfügbar.
+
+## osu!supporter kaufen
+
+::: Infobox
+![](img/store-product.jpg?2 "osu!supporter-Produktbanner aus dem osu!store")
+:::
+
+Um osu!supporter zu kaufen, besuche die [osu!supporter-Shop-Seite](https://osu.ppy.sh/store/products/supporter-tag) und passe die Dauer des Supporter-Tags mit Hilfe des Schiebereglers oder den Buttons mit der darunter befindlichen Anzahl an Monaten an. Alle Preise sind in United States Dollar (USD) und enthalten mögliche Gebühren für das Bezahlungssystem nicht.
```

Comment:
```suggestion
Um osu!supporter zu kaufen, besuche die [osu!supporter-Shop-Seite](https://osu.ppy.sh/store/products/supporter-tag) und passe die Dauer des Supporter-Tags mit Hilfe des Schiebereglers oder den Buttons mit der darunter befindlichen Anzahl an Monaten an. Alle Preise sind in United States Dollar (USD) angegeben und enthalten keine mögliche Gebühren für das Bezahlungssystem.
```

Reply:
I think this needs to be `möglichen` in this case if I'm not mistaken.

Reply:
whoops